### PR TITLE
some small changes

### PR DESCRIPTION
**Pull request by digitronik**
Signed-off-by: Nikhil Dhandre <ndhandre@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,32 @@
 # Fun with Python 3
 
-A hands on tutorial intented towards learning Python3 by solving problems.
+A hands-on tutorial intended for learning Python3 by solving problems.
 
 ## Problem Statement:
-Python Programming langauage is Open Source. It's code and all development
-happens on github.
+[Python](https://www.python.org/) is an open source programming language. It's code and all
```
Comment (chkumar246) [inline README.md]: Can we add info about Guido also?

**Comment by chkumar246** [comment]
@bhavin192 @amolkahat please have a look, Thanks :-)

**Comment by digitronik** [comment]
@chkumar246 added. have a glance.

File: README.md [hunk 2]
```diff
@@ -36,4 +37,4 @@ https://www.python.org/downloads/
 
 * [Begin your Python Journey](beginning.md)
 * [Have fun with Python Data Structures](datat_structures.md)
```
Comment (chkumar246) [inline README.md]: this link does not work, please fix it in the same pr and write in the commit message what other things you fixed.

**Comment by chkumar246** [comment]
Looks Good to me! Thanks for working on it!

**Pull request merged**